imap_processing/swapi/swapi_utils.py

```diff
@@ -0,0 +1,76 @@
+import collections
+from enum import IntEnum
+
+import xarray as xr
+
+
+class SWAPIAPID(IntEnum):
+    """Create ENUM for apid.
+
+    Parameters
+    ----------
+    IntEnum : IntEnum.
+    """
+
+    SWP_HK = 1184
+    SWP_SCI = 1188
+    SWP_AUT = 1192
+
+
+def add_metadata_to_array(data_packet, metadata_arrays):
+    """Add metadata to the metadata_arrays.
+
+    Parameters
+    ----------
+    data_packet : space_packet_parser.parser.Packet
+        SWE data packet
+    metadata_arrays : dict
+        metadata arrays.
+    """
+    for key, value in data_packet.header.items():
+        metadata_arrays.setdefault(key, []).append(value.raw_value)
+    for key, value in data_packet.data.items():
+        metadata_arrays.setdefault(key, []).append(value.raw_value)
```

Comment:
Iterate over the combination?
```suggestion
    for key, value in (data_packet.header | data_packet.data).items():
        metadata_arrays.setdefault(key, []).append(value.raw_value)
```